index.html

```diff
@@ -0,0 +1,420 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="assets/css/style.css" rel="stylesheet" type="text/css">
```

Comment:
why do you use type here, it is not according to you style guide

Reply:
fix it